source/main.lua

```diff
@@ -189,10 +189,10 @@ function love.load()
 		local flags = {fullscreen = false,display = 1,resizable = true, borderless = false}
 		love.window.setMode(gintScreenWidth, gintScreenHeight, flags)
     end
-	
+
 	local sock = require 'socket'	-- socket is native to LOVE but needs a REQUIRE
 	HOST_IP_ADDRESS = sock.dns.toip(sock.dns.gethostname())
-	sock = nil	
+	sock = nil

```

Comment:
why settings sock = nil? I get that it's just used once but I don't the the benefit of removing it again :)

Reply:
Will it free up memory? If not we can take it out.

Reply:
Like 2kb maybe? I don't think setting anything to nil should be needed. For objects we just need to make sure they get removed from tables and big assets should be freed when they are not needed, but nothing we currently have is that  big.

Reply:
Okay. Maybe Phil can delete that one line for me.

Reply:
deleted